cookbooks/aws-parallelcluster-install/templates/default/base/cfn-hup-runner.sh.erb

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+
+# This script is used to run cfn-hup is a way that is suitable to be managed by supervisord.
+# In particular, cfn-hup is executed in no-daemon mode.
+#
+# Usage:   ./cfn-hup-runner.sh
+
+[ -f /etc/profile.d/proxy.sh ] && . /etc/profile.d/proxy.sh
+
+while true; do
+  <%= node['cluster']['cfn_bootstrap_virtualenv_path'] %>/bin/cfn-hup --no-daemon --verbose
```

Comment:
If I'm not wrong cfn-hup log messages are going to cfn* log files. Can you check if cfn logs are correctly populated (and uploaded on CloudWatch) after the restart and update of the cluster?

Reply:
Confirmed: cfn logs are populated and sent to CloudWatch. 
Just to be precise: cfn-hup logs are currently not sent to CW, but only cloud-init ones